### ISSUE TITLE
SeMaWi versionsvariabel eller side

### ISSUE THREAD

**Issue by JosefAssad** [JosefAssad/SeMaWi#71]
Ville være smart at kunne se i SeMaWi hvilken version er deployeret.

**Comment by ClockEnd** [comment]
Nemlig, hænger det sammen Dockers?

**Comment by JosefAssad** [comment]
Ja, det vil klart være smart hvis Dockerfile henter version fra git tag og opretter fx en MediaWiki variabel som SEMAWI_VERSION eller lignende.
